subxt/src/rpc/rpc_client_t.rs

```diff
@@ -62,3 +62,6 @@ pub type RpcFuture<'a, T> =
 /// The inner subscription stream returned from our [`RpcClientT`]'s `subscription` method.
 pub type RpcSubscription =
     Pin<Box<dyn Stream<Item = Result<Box<RawValue>, RpcError>> + Send + 'static>>;
+
+/// The ID associated with the [`RpcClientT`]'s `subscription`.
+pub type RpcSubscriptionId = String;
```

Comment:
This is configurable in substrate via the `SubscriptionIdProvider` so it could something different than String. Should be String 99% of cases anyway but just saying.

Does the new `rpc api v2` specify the format of the subscription ID? :)

Reply:
The RPC layer should take the `subscriptionID` as a parameter in string format. From [chainHead_unstable_body](https://paritytech.github.io/json-rpc-interface-spec/api/chainHead_unstable_body.html#chainhead_unstable_body): 

> followSubscription: An opaque string that was returned by chainHead_unstable_follow.